r_src/api.rb

```diff
@@ -0,0 +1,13 @@
+class API
```

Comment:
ideally this should be compatible with rails routers, can you add a test to make sure that's the case?